apps/journeys-admin/src/components/Editor/ActionsTable/ActionsList/ActionsListView/ActionsListView.tsx

```diff
@@ -66,11 +69,13 @@ export function ActionsListView({
             <TableRow sx={{ backgroundColor: 'background.paper' }}>
               <TableCell width={60} />
               <TableCell>
-                <Typography variant="subtitle2">Target and Action</Typography>
+                <Typography variant="subtitle2">
+                  {t('Target and Action')}
+                </Typography>
               </TableCell>
               <TableCell>
                 <Typography variant="subtitle2" align="center" width={100}>
-                  Appears on{' '}
+                  {t('Appears on {{space}}', { space: ' ' })}
```

Comment:
I dont actually think we need this space
```suggestion
                  {t('Appears on)}
```